src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -21,29 +21,83 @@
         align="center"
       >
         <v-col>
-          Get shareable link
-          <v-menu
+          Share
+          <v-dialog
+            v-model="dialog"
             offset-y
-            :close-on-content-click="false"
             min-width="420"
-            max-width="420"
+            max-width="500"
           >
             <template #activator="{ on }">
               <v-icon
                 color="primary"
                 v-on="on"
               >
-                mdi-link
+                mdi-share-variant
               </v-icon>
             </template>
+
             <v-card>
-              <CopyText
-                class="pa-2"
-                :text="permalink"
-                icon-hover-text="Copy permalink to clipboard"
-              />
+              <v-card-title class="text-subtitle-2 font-weight-bold">
+                SHARE THIS ARTICLE
+                <v-btn
+                  icon
+                  x-small
+                  :right="true"
+                  :absolute="true"
+                  @click="dialog = false"
+                >
+                  <v-icon>mdi-close</v-icon>
+                </v-btn>
+              </v-card-title>
+              <v-card-title class="text-h6">
+                {{ meta.name }}
+              </v-card-title>
+              <v-card-text>
+                <div>
+                  <span
+                    v-for="author in contributors"
+                    :key="author.name + author.identifier"
+                    class="text-body-1"
+                  >
+                    {{ author.name }}
+                  </span>
+                </div>
+              </v-card-text>
+              <v-card-text>
+                <span class="font-weight-black">
+                  Share this article:
+                </span>
+                <CopyText
+                  class="pa-2"
+                  :text="permalink"
+                  icon-hover-text="Copy permalink to clipboard"
+                />
+              </v-card-text>
+              <v-divider class="mx-4" />
+              <v-card-actions>
+                <v-list-item class="grow">
+                  <v-row
+                    align="center"
+                  >
+                    <ShareNetwork
+                      network="twitter"
+                      :url="permalink"
+                      :title="meta.name"
```

Comment:
I think the the title prop here could use a bit more information. As it stands, you click on the Twitter icon and you get a very barebones tweet that just lists the title and the URL. I don't think it needs to be fixed now so I've filed #892 to track it.

Reply:
yes, this could be improved depending on what info we want to share. (i wasn't sure of the details so just included what i thought is necessary). we could also include maybe hashtags (dandi)?

Reply:
Adding a hashtag prop with `dandi` in it is a great idea. Other than that, I've deferred to #892 where we can discuss what, if anything, to do.